### DISCUSSION TITLE
DOCX->MD using --from docx+citations

### DISCUSSION THREAD

**Discussion by thomjur** [jgm/pandoc#9830] [Q&A] [answered]
Hi everyone,

I am using the pandoc/core:3.1-ubuntu container, and I am encountering several issues when converting docx files which include citations created with Zotero plugin (using `--from docx+citations`: 

1. Conversion between Author (2008) to Author [-@citekey] do not seem to be working.
2. [DELETED]

Is this a known issue? I couldn't find anything related but wanted to make sure before opening an issue.

Thanks!

**Comment by estedeahora** [comment, marked as answer]
The problem of author suppressing citations was addressed in the issue #9116 . It is a problem that requires changes to the Haskell docx reader. In that issue I propose a ( temporary ) alternative solution to solve it by a filter.

**Reply by jgm** [reply]
> Basic Word tables are no longer transformed to Markdown tables (however, they are if I am not using `--from docx+citations`).

This is certainly unexpected.  Having a small docx to test with would be helpful.

**Reply by thomjur** [reply]
Thank you both for your immediate response! I am sorry for the false alarm regarding the tables – there was an issue in one of my scripts where I was removing the embedded YAML bibliography when using docx+citations. So, nothing's wrong with pandoc here. Regarding the missing author: thanks for sharing the issue, including the proposed filter-workaround.
